### PR TITLE
chore: Update for release 0.21.0

### PR DESCRIPTION
**Pull request by rhuss**

### CODE REVIEW

**Comment by rhuss** [comment]
There is not auto-approval anymore (that has changed). @maximilien @navidshaikh could you please approve this PR ?

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -21,12 +21,21 @@
 | Add `--context` and `--cluster` flags to choose specific context from kubeconfig
 | https://github.com/knative/client/pull/1234[#1234]
 
+| ✨
+| Enable _errorlint_, _gosec_ and _unparam_ linter
+| https://github.com/knative/client/pull/1234[#1223]
```
Comment (navidshaikh) [inline CHANGELOG.adoc:26]:
```suggestion
| https://github.com/knative/client/pull/1223[#1223]
```

**Review by maximilien — APPROVED** [review]
/lgtm

**Comment by navidshaikh** [comment]
I consent.

**Comment by navidshaikh** [comment]
@googlebot I fixed it.

**Comment by navidshaikh** [comment]
@googlebot I fixed it.

**Comment by rhuss** [comment]
Thanks !

**Pull request merged**